src/plugins/launcher/launcher.ts

```diff
@@ -47,4 +47,22 @@ export class AWLauncher extends AWPlugin {
       []
     );
   }
+  registerListener(successHandler: any) {
+    AWProxy.exec(
+      successHandler,
+      (() => this.errorHandler)(),
```

Comment:
you can use `bind` instead of invoking the function immediately to preserve the scope

`this.errorHandler.bind(this)`